test/js-native-api/test_date/test_date.c

```diff
@@ -0,0 +1,65 @@
+#define NAPI_EXPERIMENTAL
+
+#include <js_native_api.h>
+#include "../common.h"
+
+static napi_value createDate(napi_env env, napi_callback_info info) {
```

Comment:
I’m not sure if we follow a convention for these names in the N-API tests, but elsewhere in the code this would be either `create_date` or `CreateDate` :)

Reply:
I was torn on this one as I saw a mix of case usage for the function names in the tests.

```
static napi_value createError(napi_env env, napi_callback_info info) {
static napi_value createPromise(napi_env env, napi_callback_info info) {
```

```
static napi_value CreateTypedArray(napi_env env, napi_callback_info info) {
static napi_value CreateDataView(napi_env env, napi_callback_info info) {
```

Reply:
I don't think we have any different convention for N-API.

Reply:
Yeah, it’s not like it matters much – the latter style would be more consistent with the rest of core, that’s all.